src/utils/constants.ts

```diff
@@ -16,6 +16,7 @@ export const SQUARELINK_ID = {
 }
  */
 export const INFURA_TOKEN = process.env.REACT_APP_INFURA_TOKEN || ''
+export const SAFE_APPS_RPC_TOKEN = process.env.REACT_APP_SAFE_APPS_RPC_TOKEN || ''
```

Comment:
Donwe have this set in out ci already?

Reply:
good catch, I've just pushed a new commit that sets this variable in the ci build and added the variable to github secrets

Reply:
@mikheevm I would add some reference to Infura in this name.

Maybe REACT_APP_SAFE_APPS_INFURA_TOKEN would be more explicit, as this token is only valid for Infura RPCs

Reply:
I didn't add infura to the name, because I was said we're not firm on infura (e.g we can switch to a different provider) and they still use same token-based authentication mechanisms

Reply:
Ok, I was referring specially to the secret set on GIthub (so it shouldn't be changed here but in Github Actions file)

You won't be able to know if the secret is storing an API key from Infura or any other provider, so I think it will be useful to change the secret name to include INFURA so you know what is stored there

Reply:
I will change the name but we should expect how we handle this info change in the future: https://github.com/gnosis/safe-config-service/issues/171

Reply:
I was just happy with changing the Github secret name, not necessarily the environment variable, but it's fine :smile: . This will most likely be deprecated once we transition this to the configuration service